inlang/source-code/plugins/icu1-importer/src/settings.ts

```diff
@@ -0,0 +1,12 @@
+import { Type, type Static } from "@sinclair/typebox"
+
+export type PluginSettings = Static<typeof PluginSettings>
+export const PluginSettings = Type.Object({
+	messagesFolderPath: Type.String({
+		pattern: ".*",
+		examples: ["./messages", "./i18n"],
+		title: "Path to folder containing language files",
+		description:
+			"The relative path to the folder containing the language files in your repository. It must contain files named `{locale}.json`.",
+	}),
```

Comment:
Why don't you use the pathPattern concept like other plugins?

- you would support different patterns like folders, files, namespaces
- aligns with other plugins

```
./messages/{locale}.json
./{locale}/file.json
```

Reply:
Until I changed the API to also pass the `settings` into `importFiles` there would have been no way to tell which file was associated with which language. Now that we have the settings I can revert back to `pathPattern`